contracts/Removal.sol

```diff
@@ -433,7 +433,7 @@ contract Removal is
       to: address(_certificate),
       ids: ids,
       amounts: amounts,
-      data: abi.encode(certificateRecipient, certificateAmount)
+      data: abi.encode(certificateRecipient, certificateAmount, address(0), 0)
```

Comment:
since this is the `migrate` function, there is no purchasing token or price multiple... but we have to pass this as data that will be decoded by `Certificate.onERC1155BatchReceived`.... what do we think about 0 address and 0 price multiplier as defaults?

Reply:
Sounds reasonable, though check my other comments about whether we even need these values at all

Reply:
yeah maybe scott can weigh in on this but I think we need this because we don't actually have the purchasing token contract address in the originating transaction because we do not use an ERC777 anymore -- it's just opaque permit sig args